code/__DEFINES/metacoin.dm

```diff
@@ -1,14 +1,14 @@
 /// Rewarded when you complete all your objectives as a traitor
 #define METACOIN_GREENTEXT_REWARD        200
 /// Rewarded when you earn an achievement
-#define METACOIN_ACHIEVEMENT_REWARD      100
+#define METACOIN_ACHIEVEMENT_REWARD      200
 /// Rewarded when you complete a crew objective
-#define METACOIN_CO_REWARD				40
+#define METACOIN_CO_REWARD				200
 /// Rewarded when you escape on the shuttle
-#define METACOIN_ESCAPE_REWARD           30
+#define METACOIN_ESCAPE_REWARD           200
```

Comment:
Way too high just for surviving a round and leaving.

Reply:
would 100 be too much?